File: unidade01/conceitos/relacional/pessoa.js
export class Pessoa {
  #nome;
  #idade;
  constructor(nome, idade) {
    this.#nome = nome;
    this.#idade = idade;
  }

  getNome() {
    return this.#nome;
  }

  getIdade() {
    return this.#idade;
  }

  setNome(nome) {
    this.#nome = nome;
  }
}

File: estudo-aoo/src/Aplicacoes/orientacaoAObjeto/Pessoa.java
package Aplicacoes.orientacaoAObjeto;

public class Pessoa {
    protected String name;
    protected int idade;

    public Pessoa(String name, int idade) {
        this.name = name;
        this.idade = idade;
    }

    // Getters e Setters

    public String getName(){
        return name;
    }

    public int getIdade(){
        return idade;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setIdade(int idade){
        this.idade = idade;
    }
}
